template/package.json

```diff
@@ -16,7 +16,7 @@
     "yetch": "^1.1.0"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.18",
+    "@types/jest": "24.0.18",
```

Comment:
Why ?

Reply:
`@testing-library/jest-dom` has bad types, when jest updated their types, they started clashing. Hopefully they fix that sometime soon, but for now, locking types.